### DISCUSSION TITLE
what is the password from the admin panel?

### DISCUSSION THREAD

**Discussion by pacanchikvgucci** [clynt707/Panel#5] [Q&A] [answered]
what is the password from the panel?  i cant decode your pass

**Comment by NotClynt** [comment, marked as answer]
By default admin data are admin, admin.

Username: admin
Password: admin

**Reply by pacanchikvgucci** [reply]
the minimum allowed number of characters is 8, current 5.
the admin password does not fit

**Reply by pacanchikvgucci** [reply]
I used your database from the description

**Reply by NotClynt** [reply]
Remove "required minlength="8" pattern="^(?!^\s.*$)(?!^.*\s$)[ -~]+$"" in line 80.
File: login.php

**Reply by pacanchikvgucci** [reply]
ty bro
